index.js

```diff
@@ -61,6 +61,43 @@ app.get('/products/:Id/description-and-standards', (req, res) => {
   });
 });
 
+app.get('/products/:searchTerm/description', (req, res) => {
```

Comment:
something to consider to modularize code. All the code is inside index.js - maybe add a controller and/or model file to modularize?